service-api/module/Application/src/Fixtures/SsmHandler.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Fixtures;
```

Comment:
This shouldn't be in the fixtures folder/namespace, but probably in `Application\Aws` instead?